packages/console/src/components/AppContent/components/Sidebar/icons/Role.tsx

```diff
@@ -0,0 +1,14 @@
+import type { SVGProps } from 'react';
+
+const Role = (props: SVGProps<SVGSVGElement>) => {
```

Comment:
@charIeszhao  are we still manually writing tsx svg component in AC?

Reply:
The existing sidebar icons are all tsx svg components.
I prefer to align with the old code and refactor them together later.

Reply:
No, we can directly import an SVG file as a component, with the help of the parcel SVG transformer plugin.

Reply:
I don't recommend aligning with the old code as the old one are going to be refactored.

Reply:
These svg components are something slipped through the net... which should've been eliminated already